public/index.html

```diff
@@ -25,12 +23,24 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
-  </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
+  <!-- Favicon images -->
+  <link rel="icon" href="https://github.com/setlife-network/website/files/3630402/favicon_io.zip">
```

Comment:
You will need to unzip the zip file and find the .ico file inside

I think this guide may help explain as well: https://medium.com/@jenniferdobak/the-public-folder-and-favicons-in-create-react-app-8dc2cc1d492b